### PR TITLE
Include app state in lookup tool (bug 1166528)

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: mkt/lookup/serializers.py [hunk 1]
```diff
@@ -2,14 +2,22 @@
...
     class Meta(ESAppSerializer.Meta):
-        fields = ['id', 'url', 'app_slug', 'name']
+        fields = ['app_slug', 'id', 'name', 'status', 'url']
 
     def get_app_summary_url(self, obj):
         return reverse('lookup.app_summary', args=[obj.id])
+
+    def get_app_status(self, obj):
+        status = mkt.STATUS_CHOICES_API_v2[obj.status]
+        if obj.status != mkt.STATUS_DISABLED and obj._is_disabled:
```
Comment (eviljeff) [inline mkt/lookup/serializers.py]: I'm pretty sure developers can't disable their app after we've Banned it (or make any changes), so this couldn't happen.  Harmless though.

File: mkt/lookup/tests/test_views.py [hunk 2]
```diff
@@ -660,6 +660,8 @@ def verify_result(self, data):
         eq_(data['objects'][0]['url'], reverse('lookup.app_summary',
                                                args=[self.app.pk]))
         eq_(data['objects'][0]['app_slug'], self.app.app_slug)
+        eq_(data['objects'][0]['status'],
+            mkt.STATUS_CHOICES_API[self.app.status])
```
Comment (eviljeff) [inline mkt/lookup/tests/test_views.py]: should this be API_V2 ?

**Comment by eviljeff** [comment]
r+wc

File: mkt/lookup/tests/test_views.py (hunk 2 above)
Comment (robhudson) [inline mkt/lookup/tests/test_views.py]: ah yeah, thanks. Probably ok here b/c it's public but better to have it right.

File: mkt/lookup/serializers.py (hunk 1 above)
Comment (robhudson) [inline mkt/lookup/serializers.py]: This is there b/c we only store `is_disabled` in elasticsearch, which comes from a method on the app object and is true when it is either `STATUS_DISABLED` or `disabled_by_user`. So this conditional is needed to pull it back apart and only return `'disabled'` if it's `disabled_by_user`.

Comment (eviljeff) [inline mkt/lookup/serializers.py]: Can we add a comment explaining?  (Took me some thought to get my head around it)
Also, nit - the lookup on #20 is redundant if the if.. is true so should be in an else instead I guess.

**Pull request merged**